CryptoAnalysis/src/main/java/crypto/reporting/CommandLineReporter.java

```diff
@@ -90,10 +107,9 @@ public void afterAnalysis() {
 		}
 		s += "=====================================================================";
 
-		System.out.println(s);
-		if (outputFolder != null) {
+		if (reportformat != null) {
 			try {
-				FileWriter writer = new FileWriter(outputFolder +"/CogniCrypt-Report.txt");
+				FileWriter writer = new FileWriter(outputFolder +"\\CryptoAnalysis-Report.txt");
```

Comment:
Please use a private constant for the file name.

Also please use `/` as a path separator as this is allowed on linux and windows while `\\` is windows only.